pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pyquil-for-azure-quantum"
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Alternatively, we could make this 1.0.0, and use 2.0.0 for a version that supports azure-quantum 2.0.

Reply:
Since we're supporting multiple major versions of `azure-quantum`, there is no need to bump this to 1.0. (In fact, it is no longer a breaking change.)

Reply:
So 0.1.1 then?

Reply:
That's the minimal version semver would permit, yes, but arguably permitting a broader range of compatibility is also a "feature" so it should have a feature bump. I don't think it matters a ton either way, especially since we have so few releases of this package.